src/pages/resultsView/enrichments/AlterationEnrichmentsContainer.tsx

```diff
@@ -70,7 +73,19 @@ export default class AlterationEnrichmentContainer extends React.Component<IAlte
     }
 
     @computed get data(): AlterationEnrichmentRow[] {
+        if (this.props.data && this.props.data.length >0) {
+            this.props.data.forEach((a:AlterationEnrichment)=>{
+            const refGenes = this.props.referenceGenes;
+            if (refGenes && refGenes.length > 0) {
+                const gene = refGenes.find(g => g.entrezGeneId === a.entrezGeneId);
+                if (gene) {
+                    a.cytoband = gene.cytoband;
```

Comment:
@adamabeshouse wanted to get your opinion here.  Seems like we are extending api data here.  One question I have is why AlterationEnrichment type has a cytoband property if it is not populated?  or are overriding it here?  It concerns me that we mutating global data source based on referenceGenes property of this component.  If this is accetable to do a general level (application wide) then we should do this in the store to ensure that all consumers of the data see the same thing under all circumstances.

Reply:
agreed. we should do it at the top.